send/generic_send

```diff
@@ -49,7 +49,8 @@ else
 	if [ -r "${PERUN_CERT}" -a -r "${PERUN_KEY}" ]; then
 		PERUN_CERT_SETTING="--cert ${PERUN_CERT} --key ${PERUN_KEY}"
 	fi
-	TRANSPORT_COMMAND="curl ${PERUN_CERT_SETTING} -i -H Content-Type:application/x-tar -f -X PUT --data-binary @- "
+	TMPFILE=`mktemp /tmp/perun-generic-send-script.XXXXXX`
```

Comment:
It's rare but still possible for mktemp command to fail. You should add check for this type of error.

Reply:
Should be fixed now.